language-plutus-core/src/Language/PlutusCore/Constant/Apply.hs

```diff
@@ -79,6 +82,10 @@ instance ( PrettyBy config (Constant ())
         [ "A non-constant argument of a sized type: "
         , prettyBy config arg
         ]
+    prettyBy config (UnreadableBuiltinConstAppError arg)    = fold
+        [ "A built-in couldn't be converted from Plutus Core to Haskell:", "\n"
```

Comment:
I'd prefer to say something like "Could not construct denotation for a built-in" or something that's less implementation-language specific.

Reply:
I've changed that, but I think the error message is now slightly more cryptic.